_config.php

```diff
@@ -69,3 +70,14 @@
 });
 
 CMSMenu::remove_menu_class(CMSProfileController::class);
+
+CMSMenu::add_link(
+    'Help',
+    function () {
+        return _t(LeftAndMain::class . '.HELP', 'Help', 'Menu title');
```

Comment:
Remember class constants can't be text collected. Best to avoid if you can for now! Sorry it's not obvious.